src/angular/schematics/ng-add/migrations/sbb-angular-symbols.json

```diff
@@ -301,6 +301,10 @@
   "SbbIconModule": "icon",
   "SbbIconOptions": "icon",
   "SbbIconRegistry": "icon",
+  "TrustedHTML": "icon",
```

Comment:
I don't think this is necessary, as these are not types that existed before

Reply:
I agree, but I thought it's not bad to let yarn sym be in sync.